benchmarks/nemo/nemolite2d/manual_versions/regent/README.md

```diff
@@ -59,3 +59,7 @@ x-ordinate, y-ordinate, depth, sea-surface height, u, v
 
 where u and v are the x and y components of velocity, respectively.
 
+## License Note##
+Most of this implementation is under the BSD 3-Clause license, unless otherwise
+noted. The Apache-2 License in this folder appliked to one of the functions in
```

Comment:
I suggest this text: "Most of this implementation is under the BSD 3-Clause license available in the repository root directory, the exception is the 'get_optional_arg' function in read_config.rg that has been copied from https://github.com/StanfordLegion/legion and is distributed with the Apache-2 licence file available in this directory."

And maybe also rename the LICENSE.txt in this directory to LICENSE.apache_v2.txt to avoid confusions.

Reply:
Done